src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -125,9 +125,18 @@
 
 						if ( $this.tribe_has_attr( 'data-tribejson' ) ) {
 
-							var data = $this.data( 'tribejson' );
+							var data = $this.data( 'tribejson' )
+							if ( 'string' === typeof data ) {
+								try {
+									data = JSON.parse( data )
```

Comment:
As above - throughout this change we should use semicolons to terminate statements